File: OKR.API/OKR.DTO/DataChart.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OKR.DTO
{
    public class DataChart
    {
        public int? OldPoint { get; set; }
        public int? NewPoint { get; set; }
        public string Label { get; set;}
        public DateTime? Date { get; set; }
        public string UserName { get; set; }
        public int ObjectivesCompletionRate { get; set; }
        public int KeyresultCompletionRate { get; set; }
    }
}

File: OKR.UI/src/Models/DataChart.ts
export class DataChart {
    oldPoint?: number; 
    newPoint?: number;
    label: string;
    date?: Date; 
    userName: string;
    objectivesCompletionRate?: number;
    keyresultCompletionRate?: number;

    constructor(
        oldPoint?: number,
        newPoint?: number,
        label: string = '',
        date?: Date,
        userName: string = ''
    ) {
        this.oldPoint = oldPoint;
        this.newPoint = newPoint;
        this.label = label;
        this.date = date;
        this.userName = userName;
    }
}